documentation/specs/proposed/BuildCheck-net10-polishing.md

```diff
@@ -0,0 +1,68 @@
+# BuildCheck Net10 polishing 
+
+In net 9.0 we delivered initial preview of BuildCheck feature. 
+ * Spec: https://github.com/dotnet/msbuild/blob/main/documentation/specs/BuildCheck/BuildCheck.md
+ * Work track:  https://github.com/orgs/dotnet/projects/373
+
+In addition to that we have item tracking possible improvemnts and extensions of the feature: https://github.com/dotnet/msbuild/issues/10548
+
+This doc focuses on minimal investment path that would still allow driving some adoption of BuildChecks feature and collecting feedback from real life usage.
```

Comment:
```suggestion
This doc focuses on a minimal investment path that would allow driving adoption of the BuildChecks feature and collecting feedback from real life usage.
```